generator/cmd/protoc-gen-gclient/main_test.go

```diff
@@ -34,4 +36,12 @@ func TestRun_Rust(t *testing.T) {
 	if err := run(os.Stdin, os.Stdout, inputPath, outDir, templateDir); err != nil {
 		t.Fatal(err)
 	}
+
+	cmd := exec.Command("cargo", "fmt", "--manifest-path", outDir+"/Cargo.toml")
```

Comment:
nit: Maybe we should expose a post-processing hook in the Codec that takes the outDir and can manipulate the sources as desired? Feels like a smell to call this from a Go test imo.